diff --git a/client/components/course/Settings/index.vue b/client/components/course/Settings/index.vue
index 4ecb0eac..fafba533 100644
--- a/client/components/course/Settings/index.vue
+++ b/client/components/course/Settings/index.vue
@@ -1,5 +1,6 @@
 <template>
   <v-container>
+    <tce-example--counter></tce-example--counter>
     <v-layout row align-start>
       <v-card>
         <sidebar
diff --git a/client/main.js b/client/main.js
index a9d8724d..7caa4403 100644
--- a/client/main.js
+++ b/client/main.js
@@ -42,20 +42,47 @@ Vue.use(Timeago, {
   }
 });

+const TCE_SERVER_PORT = 9090;
+
 const registry = new ElementRegistry(Vue);
 registry.initialize().then(() => {
   sync(store, router);
-  /* eslint-disable no-new */
-  new Vue({
-    router,
-    store,
-    el: '#app',
-    render: h => h(App),
-    provide() {
-      return {
-        $teRegistry: registry,
-        $storageService: assetsApi
-      };
-    }
+  loadTCE('tce-example', () => {
+    console.info(window.__TAILOR_CONTENT_ELEMENTS__);
+    Object.values(window.__TAILOR_CONTENT_ELEMENTS__).forEach(it => Vue.use(it));
+    /* eslint-disable no-new */
+    new Vue({
+      router,
+      store,
+      el: '#app',
+      render: h => h(App),
+      provide() {
+        return {
+          $teRegistry: registry,
+          $storageService: assetsApi
+        };
+      }
+    });
   });
 });
+
+function loadTCE(name, cb) {
+  loadStyle(`//localhost:${TCE_SERVER_PORT}/${name}.iife.css`);
+  loadScript(`//localhost:${TCE_SERVER_PORT}/${name}.iife.js`, cb);
+}
+
+function loadScript(url, cb) {
+  const script = document.createElement('script');
+  script.src = url;
+  script.async = false;
+  script.onload = () => cb(null, script);
+  script.onerror = () => cb(new Error('Error loading script!'), script);
+  return (document.body || document.documentElement).appendChild(script);
+}
+
+function loadStyle(url) {
+  const link = document.createElement('link');
+  link.rel = 'stylesheet';
+  link.href = url;
+  return document.head.appendChild(link);
+}
