diff --git a/client/components/course/Settings/index.vue b/client/components/course/Settings/index.vue
index 0ef62010..f64985da 100644
--- a/client/components/course/Settings/index.vue
+++ b/client/components/course/Settings/index.vue
@@ -1,5 +1,6 @@
 <template>
   <v-container class="mt-4">
+    <tce-example--counter />
     <v-layout row align-start>
       <v-card>
         <sidebar @action="onActionClick" :is-publishing="isPublishing" />
diff --git a/client/main.js b/client/main.js
index 6a5ece16..ab5d2b86 100644
--- a/client/main.js
+++ b/client/main.js
@@ -56,18 +56,43 @@ Vue.use(Timeago, {
 const contentPluginRegistry = new ContentPluginRegistry(Vue);
 contentPluginRegistry.initialize().then(() => {
   sync(store, router);
-  /* eslint-disable no-new */
-  new Vue({
-    router,
-    store,
-    el: '#app',
-    render: h => h(App),
-    provide() {
-      return {
-        $teRegistry: contentPluginRegistry.elementRegistry,
-        $ccRegistry: contentPluginRegistry.containerRegistry,
-        $storageService: assetsApi
-      };
-    }
+  loadContentElement('tce-example', () => {
+    console.info(window.__TAILOR_CONTENT_ELEMENTS__);
+    Object.values(window.__TAILOR_CONTENT_ELEMENTS__).forEach(it => Vue.use(it));
+    /* eslint-disable no-new */
+    new Vue({
+      router,
+      store,
+      el: '#app',
+      render: h => h(App),
+      provide() {
+        return {
+          $teRegistry: contentPluginRegistry.elementRegistry,
+          $ccRegistry: contentPluginRegistry.containerRegistry,
+          $storageService: assetsApi
+        };
+      }
+    });
   });
 });
+
+function loadContentElement(name, cb) {
+  loadStyle(`//localhost:9090/${name}.css`);
+  loadScript(`//localhost:9090/${name}.iife.js`, cb);
+}
+
+function loadScript(url, cb) {
+  const script = document.createElement('script');
+  script.src = url;
+  script.async = false;
+  script.onload = () => cb(null, script);
+  script.onerror = () => cb(new Error('Error loading script!'), script);
+  return (document.body || document.documentElement).appendChild(script);
+}
+
+function loadStyle(url) {
+  const link = document.createElement('link');
+  link.rel = 'stylesheet';
+  link.href = url;
+  return document.head.appendChild(link);
+}
